doc/source/getting_started/dsintro.rst

```diff
@@ -397,6 +397,28 @@ The result will be a DataFrame with the same index as the input Series, and
 with one column whose name is the original name of the Series (only if no other
 column name provided).
 
+From a list of dataclasses
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 1.0.0
+dataclasses as introduced in `PEP557 <https://www.python.org/dev/peps/pep-0557>`,
+can be passed into the DataFrame constructor.
+Passing a list of dataclasses is equivilent to passing a list of dictionaries. However,
+it is imperative that all values in the list are dataclasses otherwise expect a TypeError.
```

Comment:
The "however" is not fully correct (as it seems to imply this is not the case for a list of dictionaries; but it is the same for that case)

Reply:
@jorisvandenbossche  how about:

Passing a list of dataclasses is equivilent to passing a list of dictionaries. 

Please be aware, that that all values in the list should be dataclasses, mixing types in the list would result in a TypeError.

Reply:
I updated the docs with the text above, please let me know if I should change it again